src/bench/nanobench.h

```diff
@@ -2183,6 +2256,7 @@ struct IterationLogic::Impl {
                                            << ", mState=" << static_cast<int>(mState));
```

Comment:
unrelated nit: Could replace the `static_cast<double>(a.count())` with `d(a)` in the lines above?

Reply:
Ha, yes; I wrote that little `d()` helper and then forgot about it...